frontend/src/components/Calendar/WeekEvent.jsx

```diff
@@ -0,0 +1,120 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import './viewStyles.css';
+
+// const getColorForBackground = (name) => {
+//   const colorsBackground = {
+//     'A': 'rgba(255, 214, 102, 0.5)',
+//     'B': 'rgba(255, 107, 109, 0.5)',
+//     'C': 'rgba(194, 158, 239, 0.5)',
+//   };
+//   return colorsBackground[name.charAt(0).toUpperCase()] || 'rgba(0, 0, 0, 0)'; 
+// };
+
+const WeekEvent = ({ event }) => {
+  const { color, kidsName, title, start, end } = event;
+
+  const commonStyle = {
+    borderRadius: '0px 7px 7px 0px',
+  };
+  const circleStyle = {
+    ...commonStyle,
+    width: '25px',
+    height: '25px',
+    backgroundColor: color,
+    justifyContent: 'center',
+    alignItems: 'center',
+    fontSize: '18px',
+    color: 'white',
+    borderRadius: '50%',
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    marginLeft: '0px',
+    marginTop: '18px'
+  };
+
+  const titleStyle = {
+    ...commonStyle,
+    fontFamily: 'Roboto',
+    color: '#081821',
+    fontSize: '16px',
+    fontWeight: '500',
+    lineHeight: '18.75px',
+    textAlign: 'left',
+    marginLeft: '0px',
+    marginTop: '22px',
+    marginBottom: '0px'
+  };
+
+  const contentWrapperStyle = {
+    display: 'flex',
+    flexDirection: 'column',
+    marginLeft: '28px',
+    marginTop: '0px',
+  };
+
+  const timingStyle = {
+    fontSize: '12px',
+    opacity: '0.4',
+    wordBreak: 'break-all',
+    whiteSpace: 'normal',
+    fontWeight: '200',
+    lineHeight: '40px',
+    marginRight: '5px',
+  };
+
+  const boxStyle = {
+    backgroundColor: `${color}4D`,
+    borderLeft: `5px solid ${color}`,
+    height: '100%',
+    alignItems: 'center',
+    position: 'relative',
+    top: 0,
+    left: '-4px',
+    start: 'top',
+    display: 'flex'
+  };
```

Comment:
same here to move some styles into css file and leave `boxStyle`, `circleStyle`  only since we use `color` as a prop 
like so:
``` 
  const boxStyle = {
    backgroundColor: `${color}4D`,
    borderLeft: `5px solid ${color}`,
  };

  const circleStyle = {
    backgroundColor: color,
  };
```